src/se/key.rs

```diff
@@ -50,6 +50,18 @@ impl<W: Write> Serializer for QNameSerializer<W> {
         ))
     }
 
+    /// We cannot store anything, so the absence of a unit and presence of it
```

Comment:
Do you mean, we cannot store nothing?

Reply:
Yes, I mean that following consistency rules we cannot store any sign that some data is present here. The rule is "name of the type does not matter". We never use type name when serialize data (except for inferring top-level element name). Because this is a serializer for a element/attribute name, we must write something, and because there is no data to write, we return error here

Reply:
Slightly reworded this comment to make it more obvious